scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -385,13 +385,14 @@ class ScanCode(name: String, config: ScannerConfiguration) : LocalScanner(name,
         val rootLicenseFile = result["files"].singleOrNull { file ->
             val path = file["path"].textValue()
             try {
-                LICENSE_FILE_MATCHERS.any { it.matches(Paths.get(path)) }
+                file["licenses"]?.singleOrNull() != null
```

Comment:
While at it, instead of picking an arbitrary non-empty license, shouldn't we return a license expression with all licenses concatenated here?

Reply:
I agree, but this is a bigger change I did not want to make now.

Reply:
Well, ok, but could you please add a `TODO` here so we don't forget about this when starting to use license expressions throughout this file?

Reply:
Done.